### PR TITLE
Create weekly-2021-01-08.md

### PR DESCRIPTION
**Pull request by andylokandy**
Signed-off-by: Andy Lok <andylokandy@hotmail.com>

### CODE REVIEW

File: meetings/minutes/weekly-2021-01-08.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+## New features
+
+* Cpp client
+  * Launched the first working version: https://github.com/tikv/client-cpp
+
+## Test
+
+* client-rust: Test with nightly TiKV in CI: https://github.com/tikv/client-rust/pull/214
+* tidb: Add more failpoints: https://github.com/pingcap/tidb/pull/22160
+* tikv: stablize test_applied_lock_collector: https://github.com/tikv/tikv/pull/9419
```
Comment (MyonKeminta) [inline meetings/minutes/weekly-2021-01-08.md:16]:
```suggestion
* tikv: stablize test_applied_lock_collector: https://github.com/tikv/tikv/pull/9419
* tikv: Fix an issue that cdc doesn't generate rollback events when observes overlapped rollbacks https://github.com/tikv/tikv/pull/9446
```

File: meetings/minutes/weekly-2021-01-08.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  * New way to set options for transactions: https://github.com/tikv/client-rust/pull/213
+  * WIP (nrc): async commit recovery protocol.
+
+## Test
+
+* client-rust: Test with nightly TiKV in CI: https://github.com/tikv/client-rust/pull/214
+* tidb: Add more failpoints: https://github.com/pingcap/tidb/pull/22160
+* tikv: stablize test_applied_lock_collector: https://github.com/tikv/tikv/pull/9419
+* tikv: Fix an issue that cdc doesn't generate rollback events when observes overlapped rollbacks https://github.com/tikv/tikv/pull/9446
+
```
Comment (ekexium) [inline meetings/minutes/weekly-2021-01-08.md:21]:
```suggestion
* async commit: regularly run jepsen/ddl with async commit and 1PC.

```

File: meetings/minutes/weekly-2021-01-08.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+**2021-01-08**
+
+Chair: @youjiali1995 
+
+## New features
+
+* Cpp client
+  * Launched the first working version: https://github.com/tikv/client-cpp
+* Rust client
+  * New way to set options for transactions: https://github.com/tikv/client-rust/pull/213
```
Comment (andylokandy) [inline meetings/minutes/weekly-2021-01-08.md:12]: This has been post in last weekly

File: meetings/minutes/weekly-2021-01-08.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  * WIP (nrc): async commit recovery protocol.
+
+## Test
+
+* client-rust: Test with nightly TiKV in CI: https://github.com/tikv/client-rust/pull/214
+* tidb: Add more failpoints: https://github.com/pingcap/tidb/pull/22160
+* tikv: stablize test_applied_lock_collector: https://github.com/tikv/tikv/pull/9419
+* tikv: Fix an issue that cdc doesn't generate rollback events when observes overlapped rollbacks https://github.com/tikv/tikv/pull/9446
```
Comment (MyonKeminta) [inline meetings/minutes/weekly-2021-01-08.md:20]:
```suggestion
```

Sorry.. this should be in Bugfix section

File: meetings/minutes/weekly-2021-01-08.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+* async commit: regularly run jepsen/ddl with async commit and 1PC.
+
+## Bugfix
+
+* tidb: change check logic for column type change for amend: https://github.com/pingcap/tidb/pull/22150
+* tikv refactoring: remove extra op from `MvccTxn`: https://github.com/tikv/tikv/pull/9173
+* tidb: fix gc miss locks when region merged during scanning & resolving locks: https://github.com/pingcap/tidb/pull/22252
+* tikv: fix committed_cb is always invoked which affects the correctness of async apply: https://github.com/tikv/tikv/pull/9136
```
Comment (MyonKeminta) [inline meetings/minutes/weekly-2021-01-08.md:28]:
```suggestion
* tikv: fix committed_cb is always invoked which affects the correctness of async apply: https://github.com/tikv/tikv/pull/9136
* tikv: Fix an issue that cdc doesn't generate rollback events when observes overlapped rollbacks https://github.com/tikv/tikv/pull/9446
```

**Pull request merged**